doc/base/classes.xml

```diff
@@ -22137,7 +22141,7 @@
 			<argument index="0" name="id" type="int">
 			</argument>
 			<description>
-				Return a resolved item address, or an empty string if an error happened or resolution didn't happen yet (see [method get_resolve_item_status]).
+				Returns a resolved item address if one is available. Returns an empty string on error or if resolution hasn't happened yet (see [method get_resolve_item_status]).
```

Comment:
What's a resolved item address? As a developer, you'll read the name of the method, which often tells you some of what it does. The description should help you understand the details.
NB: if you explained item addresses it in the class description, it's okay - tell the user to "read the class description above for more information", or you may re-explain it in short here